examples/miscellaneous/miscellaneous_ex12/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#set -x
+
+source $LIBMESH_DIR/examples/run_common.sh
+
+example_name=miscellaneous_ex12
+
+options="-pc_type lu -pc_factor_mat_solver_package mumps"
```

Comment:
In my experience shell problems are quite sensitive and a direct solver is almost mandatory.
Any direct solver should work though.